Gopkg.toml

```diff
@@ -78,3 +78,7 @@
   non-go = true
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  name = "github.com/google/go-cmp"
```

Comment:
It would make the life of everyone easier when the deps are sorted alphabetically. This is at least what we do in our other projects.

Reply:
Sorted.